src/index.html

```diff
@@ -11,7 +11,148 @@
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="old"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="fullDate"
+          >
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+
+        <label>
+            E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="field">
+            Password:
+          <input
+            class="pass"
+            type="password"
+            name="pass"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+         Do you love cats?
+        <label>
+          <input
+            type="radio"
+            name="yes"
+            value="yes"
+          >
+            Yes
+        </label>
+        <label>
+          <input
+            type="radio"
+            name="no"
+            value="no"
+          >
+          No
+        </label>
+        <label class="field">
+            What is your favorite color?
+          <input
+            type="color"
+            name="lovecolor"
+          >
+        </label>
+        <label class="field">
+            What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+        </label>
+        <label class="field">
+            What are your favorite brand of cars?
+          <select
+              name="cars"
+              multiple
+              required
+            >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="field">
+            How do you rate our work?
+          <input
+            type="range"
+            name="ratework"
+            id="rateOurWorks"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <label>
+            Comments:
+          <textarea
+            name="comments"
+            autocomplete="off">
+          </textarea>
```

Comment:
it has space now
<img width="276" alt="image" src="https://user-images.githubusercontent.com/102411130/220053887-703696d7-30ac-4bd8-84bc-1d37ac7aea98.png">

Reply:
<img width="885" alt="image" src="https://user-images.githubusercontent.com/102411130/220054613-b14cd4af-13c9-4606-b67a-09014b329dc5.png">
checklist